### ISSUE TITLE
BandMinMaxQuantileStateful

### ISSUE THREAD

**Issue by SuperQiang-boop** [MyuLi/SST#2]
Dear author，
          Thanks for the code. In (mat_data.py) there is a line of code that is 'from utility.util  import minmax_normalize,BandMinMaxQuantileStateful', but I didn't find (BandMinMaxQuantileStateful) in util.py，thank you for your reply！

**Comment by MyuLi** [comment]
Thank you for your error report. I updated the content of utility/util. Please check whether you can generate data after the update.

**Comment by SuperQiang-boop** [comment]
Dear author，
    When I trained the WDC dataset with sigma=25, the first epoch model converged, but the PSNR was negative, have you ever had a similar situation？Thanks for your reply!

**Comment by MyuLi** [comment]
I also encountered this situation during early training, which should be related to the calculation method of psnr during training in the code. Generally, it can quickly become a positive number. In addition, different initialization methods may also have some impact.

**Comment by SuperQiang-boop** [comment]
请问你的训练集生成多少64x64x31的patch块？

**Comment by MyuLi** [comment]
icvl数据集上是53000个patch

**Comment by SuperQiang-boop** [comment]
您好，再次打扰了。请问测试集的话是用的icvl_test.txt里的数据吗？裁剪为512x512x31是怎样裁剪的呀？感谢回复！

**Comment by MyuLi** [comment]
是的。论文的结果是使用中心裁剪的方式得到的，但室内场景裁剪出来有一些重复。可以将室内场景部分的图在中下部分进行裁剪，其余图居中裁剪。测试结果上来看，没有较大区别

**Comment by SuperQiang-boop** [comment]
您好，打扰了，在简单的测试模型时发现小一些的图像(128x128)比大一些的图像(256x256)的测试效果好，这是为啥呀？那在测试模型512x512跑不了的情况下，可以将其裁剪为256x256的图像进行测试吗？
感谢您的回复
